packages/sanity/src/ui-components/menuGroup/MenuGroup.tsx

```diff
@@ -11,7 +11,8 @@ export type MenuGroupProps = Pick<UIMenuGroupProps, 'icon' | 'popover' | 'text'
  * @internal
  */
 export const MenuGroup = (
-  props: MenuGroupProps & Omit<HTMLProps<HTMLDivElement>, 'as' | 'height' | 'ref' | 'tabIndex'>,
+  props: MenuGroupProps &
+    Omit<HTMLProps<HTMLDivElement>, 'as' | 'height' | 'ref' | 'tabIndex' | 'popover'>,
```

Comment:
`popover` is now a valid html attribute, `@sanity/ui` also had to add it to the omit: https://github.com/sanity-io/ui/commit/98c43b1ee9d5f8330786b20cd5fcda125747c509